Automattic/woocommerce-payments
# Update click tooltip styles to match design library

Fixes #6917

#### Changes proposed in this Pull Request

This PR updates the style of the [`ClickTooltip`](https://github.com/Automattic/woocommerce-payments/blob/35b5b303adfbac06fa2be5a7e6c2635102a6c6de/client/components/tooltip/index.tsx#L79) tooltip element to match the design of the Kofpo8jj4zDWXDsVdlSz0f-fi-6795%3A10109.

<img width="296px" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/be066a70-764b-441e-bae5-e1d4fd46977a" alt="tooltip design" />


**Notes**
* the ClickTooltip component is currently only used for the Account Balances Tooltip Panel ([98c2911/client/components/account-balances/index.tsx](https://github.com/Automattic/woocommerce-payments/blob/98c291141b41f87213f480a4ce271260fef29fb8/client/components/account-balances/index.tsx)), but will be used as part of the new Transaction/Dispute details screen being introduced in #6883.
* For `ClickTooltip`, I've removed the arrow that appears for `HoverTooltip` to match the design.
* I've refactored `HoverTooltip` and `ClickTooltip` to accept an optional `buttonIcon` prop, similar to [BannerNotice](https://github.com/Automattic/woocommerce-payments/blob/e6f547d53410febbc11416656afc7e72a571e1ea/client/components/banner-notice/index.tsx). This allows us to keep the color/tab/accessibility logic abstracted within the tooltip component.
* The tooltip button icon and any tooltip content links will be coloured as the _WP admin theme color_ or _Gutenberg Blue_



#### Testing instructions





* View the **Payments → Overview** screen in WP-admin.
* Click on the help icon next to `Available funds` or `Pending funds` to view the tooltip content.
* Ensure the tooltip content matches the design with a white background.

**Before**
![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/58b21942-c2f2-48e3-b703-cd2f4f7d3217)

**After**
![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/c3438e1c-fe53-475c-9f28-98bbf47fea7f)


https://github.com/Automattic/woocommerce-payments/assets/3147296/572774ad-ccb8-4561-9b4a-30ee7d68298e


**Using a custom WP admin theme color (WooExpress)**
<img width="699" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/a77584cb-7279-49d9-b546-a1cf40fee7fb">



-------------------

- [x] Covered with tests (or have a good reason not to test in description ☝️) Changes to CSS styles – testing not suitable as per current testing strategy
- [x] Tested on mobile – tested via Safari on iOS 16.6

**Post merge**
